### ISSUE TITLE
Add a feature to share images.

### ISSUE THREAD

**Issue by thisisamank** [thisisamank/resizrr#11]
When add an option near download image of home screen to share image. Once that is clicked image should be shareable to different platforms.

**Comment by ShivamMenda** [comment]
Hey @thisisamank Is this issue applicable for Hacktoberfest 2021? If it is I would like to work on this issue. I'm not sure whether I would be able to implement it or not. But I would like to try .

**Comment by ShivamMenda** [comment]
I would be using the share package and the path provider package.

**Comment by thisisamank** [comment]
yes @ShivamMenda this project is accepted at hacktoberfest. Sure, go ahead and give it a try! Best of luck :)

**Comment by ShivamMenda** [comment]
Okay will get back to you as soon as possible.Thanks.

**Comment by ShivamMenda** [comment]
@thisisamank I have opened a pull request on the share feature. Waiting for your feedback and validation. Thank you.
